### PR TITLE
Fix creating translations with app

### PR DESCRIPTION
**Pull request by krzysztofwolski**
I want to merge this change because apps were not able to create translations.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [x] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 ## [Unreleased]
+- Fix creating translations with app - #6804 by @krzysztofwolski
```
Comment (maarcingebala) [inline CHANGELOG.md:6]: Please move this entry to the "Other" section below, this is where we keep all entries that are not breaking changes.

Comment (krzysztofwolski) [reply CHANGELOG.md]: Done

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/translations/mutations.py [hunk 2]
```diff
@@ -308,6 +315,7 @@ class Meta:
         permissions = (SitePermissions.MANAGE_TRANSLATIONS,)
         error_type_class = TranslationError
         error_type_field = "translation_errors"
+        permissions = (SitePermissions.MANAGE_TRANSLATIONS,)
```
Comment (fowczarek) [inline saleor/graphql/translations/mutations.py:318]: Duplicated line. (L315)

Comment (krzysztofwolski) [reply saleor/graphql/translations/mutations.py:317]: Oooops. Fixed!

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**